hack/experiment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: redskyops.dev/v1beta1
+kind: Experiment
+metadata:
+  name: ci
+spec:
+  parameters:
+  - name: something
+    min: 50
+    max: 1000
+    baseline: 100
+  - name: another
+    min: 16
+    max: 512
+    baseline: 64
+  metrics:
+  - minimize: true
+    name: cost
+    port: 9090
+    query: ({{ cpuRequests . "redskyops.dev/trial-role=trialResource" }} * 1) + ({{ memoryRequests . "redskyops.dev/trial-role=trialResource" | GB }} * 1)
+    type: prometheus
+  trialTemplate:
+    spec:
+      approximateRuntime: 45s
+      setupServiceAccountName: redsky-setup
+      setupTasks:
+      - args:
+        - prometheus
+        - $(MODE)
+        name: monitoring
```

Comment:
Im debating adding in a trial pod here. 
Something like containers: [ locust, nginx ] so we could add a p95 latency metric just to increase the touch points. 

Thoughts?

Reply:
I'll take you one further: should we use experiment generation instead of having the experiment committed?

Reply:
I'll take you one further: should we use experiment generation here?